elephant_vending_machine/views.py

```diff
@@ -1,9 +1,11 @@
-"""Define all routes for the behavioral experiment server.
+"""
```

Comment:
Summary line should be on same line as the three quotes per [google docstring spec](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html).